### PR TITLE
Add Initial Documentation

### PR DESCRIPTION
**Pull request by cmbirk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# datatable
\ No newline at end of file
+![Linting](https://github.com/crossroadscx/datatable/actions/workflows/linting.yml/badge.svg) ![Chromatic](https://github.com/crossroadscx/datatable/actions/workflows/chromatic.yml/badge.svg)
+# React Table DataTable
+
+This project layers a UI on top of the every popular [React Table](https://react-table.tanstack.com/) library. The intention is to have a drop-in UI solution for most table use cases, while including simple options to override things like theming, sub-components, and actions.
```
Comment (Blakidarkness) [inline README.md:4]: May be we can additional info like:

- We are using our own CSS? 
- CSS frameworks can be used with this component?
- Has more dependencies besides `react table?`

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+## Demo
+
```
Comment (Blakidarkness) [inline README.md:6]: Would be nice if we add some screenshots of the component and may be some styling applied!.

**Review by Emmaalara — APPROVED** [review]
Awesome!

**Review by Blakidarkness — APPROVED** [review]
Looks great! I only added a few suggestions.

File: README.md (hunk 2 above)
Comment (cmbirk) [reply README.md:6]: Absolutely agree. We don't have the theming capability built in yet, but will include examples when that comes in. I will add some screenshots

File: README.md (hunk 1 above)
Comment (cmbirk) [reply README.md:4]: Great suggestion. Will add

File: README.md (hunk 2 above)
Comment (cmbirk) [reply README.md:6]: Resolved in 990c310d84e844d64909725416c51acd2cd89787

File: README.md (hunk 1 above)
Comment (cmbirk) [reply README.md:4]: Resolved in 990c310d84e844d64909725416c51acd2cd89787

It does have more dependencies than react-table, but they can see those in the `package.json` file. I don't think we need to include those explicitly here. The reason for including `React Table` is that this is a light UI layer on top of the hooks that library provides. It's the main engine for the table.

**Review by Blakidarkness — APPROVED** [review]
Approved. I think it's ready! 
Also I forgot on my previous comment that we can add if it's typescript friendly.

**Pull request merged**